tests/mir-opt/inline/inline_diverging.rs

```diff
@@ -1,6 +1,8 @@
 // Tests inlining of diverging calls.
 //
 // EMIT_MIR_FOR_EACH_PANIC_STRATEGY
+// compile-flags: -Zinline-mir-hint-threshold=1000
+// ignore-wasm32-bare compiled with panic=abort by default
```

Comment:
hmm... `EMIT_MIR_FOR_EACH_PANIC_STRATEGY` tries to build for `unwind`, too? Or what is the issue here?

Reply:
There is no issue, I must have goofed the rebase. Thanks for catching it!